examples/extensions/update_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,134 @@
+        "CampaignExtensionSettingOperation", version="v6"
+    )
+    extension_feed_item_service = client.get_service(
+        "ExtensionFeedItemService", version="v6"
+    )
+
+    extension_feed_items = map(
+        lambda feed_item_id: extension_feed_item_service.extension_feed_item_path(
+            customer_id, feed_item_id
+        ),
+        feed_item_ids,
+    )
+    campaign_extension_setting = campaign_extension_setting_operation.update
+    extension_type_enum = client.get_type("ExtensionTypeEnum", version="v6")
+    composite_id = ResourceName.format_composite(
+        # the docs don't make it clear that this should reference the name of the extension type and not the enum
+        # e.g. just extension_type_enum.ExtensionType.SITELINK won't work, but the string 'SITELINK' will
+        campaign_id, extension_type_enum.ExtensionType.Name(
+            extension_type_enum.ExtensionType.SITELINK
+        )
+    )
+    resource_name = campaign_extension_setting_service.campaign_extension_setting_path(
+        customer_id, composite_id
+    )
+    campaign_extension_setting.resource_name = resource_name
+    campaign_extension_setting.extension_feed_items[:] = extension_feed_items
+
+    # Produce a field mask enumerating the changed fields
+    fm = protobuf_helpers.field_mask(None, campaign_extension_setting)
+    campaign_extension_setting_operation.update_mask.CopyFrom(fm)
+
+    try:
+        response = campaign_extension_setting_service.mutate_campaign_extension_settings(
+            customer_id, [campaign_extension_setting_operation]
+        )
+        print(
+            "Updated campaign extension setting with resource name: "
+            f'"{response.results[0].resource_name}".'
+        )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+if __name__ == "__main__":
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description=(
+            "Updates the sitelink campaign extension setting to replace its "
+            "extension feed items."
+        )
+    )
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument(
+        "-c",
+        "--customer_id",
+        type=str,
+        required=True,
+        help="The Google Ads customer ID",
+    )
+    parser.add_argument(
```

Comment:
should include a short form argument, ie. `"-i"`

Reply:
ack